code/lib/csf-tools/src/ConfigFile.ts

```diff
@@ -191,16 +191,19 @@ export class ConfigFile {
             ) {
               let exportObject = right;
               if (t.isIdentifier(right)) {
-                exportObject = _findVarInitialization(right.name, parent as t.Program);
+                exportObject = _findVarInitialization(right.name, parent as t.Program) as any;
               }
               if (t.isObjectExpression(exportObject)) {
                 self._exportsObject = exportObject;
-                exportObject.properties.forEach((p: t.ObjectProperty) => {
+                (exportObject.properties as t.ObjectProperty[]).forEach((p) => {
                   const exportName = propKey(p);
                   if (exportName) {
-                    let exportVal = p.value;
+                    let exportVal = p.value as t.Expression;
```

Comment:
Why do we need all these extra type annotations?

Reply:
Yes, #20877

Reply:
@vatcoop & I are affected, can't do changes without disabling strict mode and do few changes